Bug: Error message for 'continue' outside loop incorrectly says 'break'

## Description

When using a `continue` statement outside of a loop, the error message incorrectly refers to "break" instead of "continue".

## Steps to Reproduce

```ez
import @std
using std

do main() {
    continue
}
```

## Expected Behavior

Error message should say:
```
error[E5009]: continue statement outside loop
```

## Actual Behavior

Error message says:
```
error[E5009]: continue statement outside loop
  --> file.ez:6:5
   |
6 |     continue
   |     ^ break not inside loop    <-- Should say "continue not inside loop"
```

The error code and title are correct, but the detail message says "break not inside loop" when it should say "continue not inside loop".

## Environment

- EZ version: v0.39.3
- OS: macOS

--- a/pkg/typechecker/typechecker.go
+++ b/pkg/typechecker/typechecker.go
@@ -1628,7 +1628,7 @@ func (tc *TypeChecker) checkStatement(stmt ast.Statement, expectedReturnTypes []
 		// Check that continue is inside a loop (#603)
 		if tc.loopDepth == 0 {
 			tc.addError(
-				errors.E5009,
+				errors.E5010,
 				"continue statement outside loop",
 				s.Token.Line,
 				s.Token.Column,

